podium-gateway/src/main/java/nl/thehyve/podium/service/RequestService.java

```diff
@@ -246,18 +252,32 @@ public void deleteDraft(IdentifiableUser user, UUID uuid) throws ActionNotAllowe
         log.debug("Submitting request : {}", uuid);
 
         List<Request> organisationRequests = new ArrayList<>();
+        // TODO: Aggregate mails for multiple organisations per user.
         for (UUID organisationUuid: request.getOrganisations()) {
-            // TODO: Fetch organisation DTO through Feign.
-            // OrganisationDTO organisation;
-            // OR: leave this to the (asynchronous) mail service call
-            // to notify the organisations.
+            // Fetch organisation and organisation coordinators through Feign.
+            OrganisationDTO organisation;
+            List<UserRepresentation> coordinators;
+            try {
+                organisation = organisationService.findOrganisationByUuid(organisationUuid);
```

Comment:
Could be made more clear that this uses a Feign client. Use organisationClientService